client/src/components/RightSide/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import styles from "../../App.module.css";
+import { JavaProject } from "../../JavaProjectTypes";
+import Island from "../Island";
+import DomesticDependencyArrow from "../DependencyArrow/DomesticDependencyArrow";
+import ForeignDependencyArrow from "../DependencyArrow/ForeignDependencyArrow";
+import { TransformWrapper, TransformComponent } from "react-zoom-pan-pinch";
+import { JavaArchipelago } from "../../lib/JavaArchipelago";
+import { legendWidth } from "../../util/constants";
+
+const RightSide = ({ javaProject }: { javaProject?: JavaArchipelago }) => {
+  const islands = javaProject.islands;
+  const links = javaProject.links;
+
+  return (
+    <div
+      className={styles.rightSide}
+      style={{
+        height: `${javaProject.height + 150}px`,
```

Comment:
where does the 150 come from? Might be nice to use a constant, not necessary though I guess :P (same with the 150's below)

Reply:
Yeah that's not very nice... I just wanted to add some px to the height and width to make the right side looks nicer, then ended up changing in a few places. It's not desirable but maybe just leave them there for now :P